### DISCUSSION TITLE
Databases supported

### DISCUSSION THREAD

**Discussion by yogeshkodei** [prisma/prisma#2016] [Q&A] [answered]
Can any database be used or is it limited to just the 3 listed in the documentation?

**Comment by pantharshit00** [comment, marked as answer]
Right now we support three databases:
- MySQL
- SQLite
- PostgreSQL

Support for more database will come in the future.

**Reply by nikolasburk** [reply]
Note that the full list of databases is also in the [docs](https://www.prisma.io/docs/more/supported-databases).

**Comment by ajwerner** [comment]
Have you tried whether it works with CockroachDB? CockroachDB uses the postgres wire protocol. There are some SQL differences but on the whole it's become pretty compatible.

**Comment by ezsper** [comment]
I would really appreciate some initial documentation on how to contribute to add support to other databases, I'm specially interested on Cassandra support. It will certainly not be easy to include Cassandra as it will not support some features like unique keys and its composition keys (partition and cluster) will certainly be a challenge to implement among clustering order and static columns. But, I have plenty of experience with Cassandra and the time and expertise to do this datasource inclusion for it. I believe that Cassandra makes totally sense to be part of this project, and migration for Cassandra with CQL should be totally feasible. The Cassandra community, specially the ones that are focusing on JavaScript, would benefit a lot on this project. I truly believe this is one of the most important projects right now for JavaScript on back-end.

**Reply by janpio** [reply]
A feature request for Cassandra support would be a great first step.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
